### DISCUSSION TITLE
help with missing plugin

### DISCUSSION THREAD

**Discussion by bethropolis** [Rafostar/clapper#471] [Q&A] [answered]
I have been trying to troubleshoot this gstreamer missing plugin error for hours.

I have install all the gstreamer libs the good, bad and the ugly one too.

I am using fedora 40

and this is my debug info:
```
GLib 2.80.2
GTK 4.14.4
Adwaita 1.5.1
GStreamer 1.24.4
```

here is a screenshot of the error message:

![Screenshot from 2024-07-03 22-55-57](https://github.com/Rafostar/clapper/assets/66518866/4a93ea8e-1567-4828-a594-5dc4de86e448)

**Comment by Rafostar** [comment, marked as answer]
Any reason why not just use Clapper Flatpak package? :smile: 

If you want Fedora RPM package version instead, you have to get few things yourself.
Follow: https://rpmfusion.org/Howto/Multimedia

Mainly steps:
1) Switch to full ffmpeg
2) Install Intel/AMD hardware acceleration

Also, make sure you have `gstreamer1-plugin-libav` installed.

**Reply by bethropolis** [reply]
this is a flatpak package, 
I installed from the gnome store

the fedora linux registry to be specific

**Reply by bethropolis** [reply]
nvm, I just uninstalled it and installed the flathub one and everything works 😊

thank you so much, i like the app

**Reply by Rafostar** [reply]
The Flatpak package I personally maintain is from Flathub.

**Comment by LSeelig** [comment]
I am having this issue with the Flathub flatpak and rpm, but only with peertube videos.

**Reply by Rafostar** [reply]
For PeerTube, replace URI scheme `https` -> `peertube` in link you want to play. At least Flatpak package supports this. This is to explain to the player that such site/IP hosts peertube instance and should be treated as such.

**Reply by LSeelig** [reply]
Ok. I can do this, but my use case is playing peertube videos from Pipeline, which cannot be customized to do so. maybe I could script the command?

**Reply by Rafostar** [reply]
Alternatively, create `peertube_hosts` text file in `$HOME/.var/app/com.github.rafostar.Clapper/config/gtuber-0.0` directory with hostnames that you use (one per line). Then `https` URIs will be figured out from these too.

**Reply by LSeelig** [reply]
> Alternatively, create `peertube_hosts` text file in `$HOME/.var/app/com.github.rafostar.Clapper/config/gtuber-0.0` directory with hostnames that you use (one per line). Then `https` URIs will be figured out from these too.

Thank you. I'll try it.

**Comment by igekaratas** [comment]
<img width="681" height="375" alt="Screenshot_20251206_220718" src="https://github.com/user-attachments/assets/afe07f3a-5952-4057-b81c-96758683c11d" />

I have this issue even though I installed from Flathub.

**Reply by Rafostar** [reply]
Hi, this is a different issue. Yours is that demuxer found some unknown/unsupported subtitles stream in file.
This needs either:
1. Adding support for such format upstream in GStreamer so they will work
2. Making Clapper ignore such streams to let video at least play without them.

For the 2nd option there is already open issue: https://github.com/Rafostar/clapper/issues/440
